transformers/test_data/vow_file.go

```diff
@@ -51,17 +52,16 @@ var VowFileHeaderSyncLog = core.HeaderSyncLog{
 	Transformed: false,
 }
 
-var VowFileModel = shared.InsertionModel{
-	SchemaName: "maker",
-	TableName:  "vow_file",
-	OrderedColumns: []string{
-		constants.HeaderFK, "what", "data", constants.LogFK,
+var VowFileModel = event.InsertionModel{
+	SchemaName: constants.MakerSchema,
+	TableName:  constants.VowFileLabel,
+	OrderedColumns: []event.ColumnName{
+		constants.HeaderFK, constants.WhatColumn, constants.DataColumn, constants.LogFK,
```

Comment:
Should use the constants from `event` in core for non domain specific things :)
```go
const LogFK ColumnName = "log_id"
const AddressFK ColumnName = "address_id"
const HeaderFK ColumnName = "header_id"
```

They still exist in both repos to allow for step-by-step migration of the transformers. No biggie either, we'll notice breakage when the `vdb-mcd-transformers` code is nuked anyway :tada: